playbooks/inventory.yaml

```diff
@@ -103,23 +103,24 @@
 
 
         - name: Upload inventory - {{ ansible_hostname }}
```

Comment:
This name belongs to the task not the block.

The block will need a name however, possibly "Capture error and display trace information"